hedera-mirror-rest/controllers/contractController.js

```diff
@@ -1074,17 +1093,34 @@ class ContractController extends BaseController {
     transaction,
     contractLogs,
     contractStateChanges,
-    fileData
+    fileData,
+    shouldMockContractResults
   ) => {
     res.locals[constants.responseDataLabel] = new ContractResultDetailsViewModel(
       contractResult,
       recordFile,
       transaction,
       contractLogs,
       contractStateChanges,
-      fileData
+      fileData,
+      shouldMockContractResults
     );
   };
+
+  getMockedContractResultByTransaction = (transaction) => {
+    return {
+      bloom: Buffer.alloc(256),
+      callResult: null,
+      createdContractIds: [],
+      functionParameters: [],
+      payerAccountId: transaction.payerAccountId,
+      errorMessage: TransactionResult.getName(transaction.result),
+      consensusTimestamp: transaction.consensusTimestamp,
+      contractId:
+        transaction.toAddress && transaction.toAddress.toString('hex') ? transaction.toAddress.toString('hex') : null,
```

Comment:
nit: pull out toAddress conversion
```
const toAddress = transaction.toAddress ? transaction.toAddress.toString('hex') : null;
```
```suggestion
        toAddress ? toAddress : null,
```

Reply:
`toAddress` is now replaced with `entityId`